src/SayMore/Model/ArchivingHelper.cs

```diff
@@ -21,6 +21,7 @@ namespace SayMore.Model
 {
 	static class ArchivingHelper
 	{
+		internal static IMDIPackage _Package;
```

Comment:
Couldn't this be done without adding a static member? Either passing in the _Package to methods that would add it or adding it in the calling code?